proxy/templates/scripts.js

```diff
@@ -0,0 +1,17 @@
+module.exports = (items) => `
+  <script src="/lib/react.development.js"></script>
```

Comment:
I am guessing that these are local copies of the .js library files needed? So they arn't worth loading and looking at right?

Reply:
Yes exactly, I replicated what Fred did and load the react files from disk instead of an http call

Reply:
Cool! I just wanted to make sure.